mne/utils/docs.py

```diff
@@ -383,11 +383,29 @@
 cbar_fmt : str
     String format for colorbar values.
 """
-docdict["topomap_mask"] = """
-mask : ndarray of bool, shape (n_channels, n_samples) | None
-    The channels to be marked as significant at a given time point.
-    Indices set to ``True`` will be considered. Defaults to ``None``.
-"""
+mask_base = """
+mask : ndarray of bool, shape %(shape)s | None
+    Array indicating channel%(shape-appendinx)s to highlight with a distrinct
+    plotting style%(example)s. Array elements set to ``True`` will be plotted
+    with the parameters given in ``mask_params``. Defaults to ``None``,
+    equivalent to an array of all ``False`` elements.
+"""
+topomap_mask_dstr = {
+    'shape': '(n_channels, )',
+    'shape-appendinx': '',
+    'example': ''}
```

Comment:
I would prefer compressing these dictionary lines, but I leave it up to you and @drammock to decide.

Reply:
I made it a bit more compact, but it's a pain to keep it readable in the limit of 79 characters.